### ISSUE TITLE
when casting buttons to TComponent, the tag field is always 0

### ISSUE THREAD

**Issue by marcuscde** [EtheaDev/StyledComponents#34]
The buttons are not returning the correct 'tag', when they are casted to TComponent or TWinControl.

**Comment by carloBarazzetta** [comment]
Sorry, you can fix adding those two lines:
```
procedure TCustomStyledGraphicButton.SetTag(const AValue: Integer);
begin
  inherited Tag := AValue; //add this one
  FRender.Tag := AValue;
end;
```

```
procedure TCustomStyledButton.SetTag(const AValue: Integer);
begin
  inherited Tag := AValue; //add this one
  FRender.Tag := AValue;
end;
```
It will be fixed in the next release...

**Comment by marcuscde** [comment]
Thanks it works

**Comment by marcuscde** [comment]
There was also code my boss pushed to fix the sorting of the menu bar, it was going out of order, he is with CDE software as well. Could you also integrate that fix into the next release?

**Comment by carloBarazzetta** [comment]
Fixed in 3.6.3 version.
The code for Sorting Order of StyledToolButton was added, with some differences
